__tests__/estimates.test.ts

```
@@ -219,5 +219,26 @@ describe('Estimate API', () => {
 			expect(searchResponse.results.length).toBe(estimatesInDateRange.length);
 			expect(searchResponse.results[0].Id).toBe(estimatesInDateRange[0].Id);
 		});
+
+		// Describe the getUpdatedEstimates Method
+		it('should return an empty array if no estimates are updated', async () => {
+			// Setup the Estimate Query Response
+			const estimateQueryResponse: { QueryResponse: {}; time: string } = {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)